cluster/aws/util.sh

```diff
@@ -721,14 +723,11 @@ function create-iam-profile {
 
 # Creates the IAM roles (if they do not already exist)
 function ensure-iam-profiles {
-  aws iam get-instance-profile --instance-profile-name ${IAM_PROFILE_MASTER} || {
```

Comment:
Sounds good.

And these things should be configured to have test specific values (strings) in test configs.